model/project_parser.go

```diff
@@ -1130,7 +1132,11 @@ func evaluateBVTasks(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluator, vse
 				isGroup = true
 			}
 		}
-		if err1 != nil && err2 != nil {
+		if pbvt.TaskGroup != nil {
+			names = append(names, pbvt.Name)
+			isGroup = true
+		}
+		if err1 != nil && err2 != nil && !isGroup {
```

Comment:
Is the thought that err1 and err2 are going to be not nil if pbvt.TaskGroup != nil? Because in that case we just shouldn't check those cases.